src/tsid.cpp

```diff
@@ -108,6 +108,12 @@ void setup() {
 	EEPROM.get(EEPROM_ADDR_COOKIE, value);
 	bool initialize_memory = (value != EEPROM_COOKIE_VALUE);
 
+	// are we holding reset (to reset globals)?
```

Comment:
@twisted-electrons Note that this will run the full upgrade, assuming the memory is not formatted/initialised before (or coming from a pre 2.0-firmware etc). When upgrading from V1 to V2 it will for instance change all default values on filter enable to on (for all three channels), overwriting possible patches which probably is not what is intended.

So I would suggest to only run the "Update all global settings from EEPROM memory" section (currently row 142) on boot-with-RESET-button.